tilt/Tiltfile

```diff
@@ -282,3 +287,11 @@ k8s_resource(
     objects=['pod-delete:stage'],
     resource_deps=['kwok', 'kwok-stage-fast']
 )
+
+# Incluster file server - optional component for log collection
+# Make it wait for cert-manager like other resources (even though it doesn't need certs itself,
```

Comment:
I don't follow, if it doesn't need the cert manager resources, why does it need to wait for it?